clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2737,6 +2754,16 @@ void SYCLIntegrationHeader::emit(raw_ostream &O) {
 
   O << "\n";
 
+  int cpp_version = getCppVersion();
```

Comment:
Please follow llvm coding guidelines.
See https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
Updated